### DISCUSSION TITLE
API 문서 페이지 관련 제보

### DISCUSSION THREAD

**Discussion by bangrr** [solvedac/unofficial-documentation#76] [일반]
안녕하세요, Issues 에 남기기에는 부담스러워서 Discussions에 적어봅니다.

1. SCHEMAS 카테고리 하위 페이지들에서  (go to ref) 버튼을 누르면 이동되는 페이지가 404 Not Found 발생합니다.
발생페이지: [schemas/Badge](https://solvedac.github.io/unofficial-documentation/#/schemas/Badge)
-> https://solvedac.github.io/#/schemas/${clickedUrl} 로 이동됨

2. Tier 이미지가 제대로 표시되지 않습니다.
발생페이지: [schemas/ArenaTier](https://solvedac.github.io/unofficial-documentation/#/schemas/ArenaTier)
-> GET https://solvedac.github.io/unofficial-documentation/null 404 (Not Found)

위와 같은 현상이 발생해서 말씀드립니다!

**Comment by RanolP** [comment]
아래 PR 코멘트에 있는 프리뷰 링크에서 버그가 고쳐졌다면 머지 후 토론 닫을게요

- https://github.com/solvedac/unofficial-documentation/pull/77

**Comment by RanolP** [comment]
응답이 없으셔서 그냥 머지하고 완료처리 합니다.

**Reply by bangrr** [reply]
앗, 제가 프리뷰에서 확인해야되는건줄 몰랐습니다. 고생하셨습니다!
